package.json

```diff
@@ -40,30 +41,45 @@
   },
   "homepage": "https://github.com/legokichi/ts-ebml#readme",
   "dependencies": {
-    "buffer": "^5.0.7",
-    "commander": "^2.11.0",
-    "ebml": "^2.2.1",
-    "ebml-block": "^1.1.0",
-    "events": "^1.1.1",
-    "int64-buffer": "^0.1.9",
-    "matroska": "^2.2.3"
+    "buffer": "^6.0.3",
+    "commander": "^9.3.0",
+    "ebml": "^3.0.0",
+    "ebml-block": "^1.1.2",
+    "events": "^3.3.0",
+    "int64-buffer": "^1.0.1",
+    "matroska-schema": "^1.0.0"
   },
   "devDependencies": {
-    "@types/commander": "^2.9.1",
-    "@types/qunit": "^2.0.31",
-    "browserify": "^13.1.0",
-    "empower": "^1.2.3",
+    "@playwright/test": "^1.23.2",
+    "@types/commander": "^2.12.0",
+    "@types/node": "^18.0.3",
+    "@types/qunit": "^2.19.1",
+    "aliasify": "^2.1.0",
+    "browserify": "^17.0.0",
+    "empower": "^1.3.1",
     "espower-cli": "^1.1.0",
-    "power-assert": "^1.4.4",
+    "http-server": "^14.1.1",
+    "power-assert": "^1.6.1",
     "power-assert-formatter": "^1.4.1",
     "qunit-tap": "^1.5.1",
     "qunitjs": "^2.4.0",
-    "tslint": "^5.13.1",
-    "typedoc": "^0.14.2",
-    "typescript": "^3.3.3333",
-    "watchify": "^3.7.0"
+    "tslint": "^5.20.1",
+    "typedoc": "^0.23.5",
+    "typedoc-plugin-missing-exports": "^0.23.0",
+    "typescript": "^4.7.4",
+    "watchify": "^4.0.0"
   },
   "bin": "./lib/cli.js",
   "main": "./lib/index.js",
-  "typings": "./lib/index.d.ts"
+  "typings": "./lib/index.d.ts",
+  "browserify": {
+    "transform": [
+      "aliasify"
+    ]
+  },
+  "aliasify": {
+    "aliases": {
+      "ebml": "ebml/lib/ebml.js"
```

Comment:
I just wanted to point out that this is important for consumers of `ts-ebml` too. The dependency `ebml` 3.0.0 has a `browser` entry in its package.json that points at an IIFE bundle which is not what bundlers expect so it has to be aliased to `ebml/lib/ebml.js` or `ebml/lib/ebml.esm.js`